it/uk/gov/hmrc/agentmapping/MappingControllerISpec.scala

```diff
@@ -502,36 +224,41 @@ class MappingControllerISpec extends UnitSpec with MongoApp with WireMockSupport
 
         givenUserNotAuthorisedWithError("MissingBearerToken")
         givenIndividualRegistrationExists(utr)
-        createMappingRequest(saOnlyIdentifier).putEmpty().status shouldBe 401
+        createMappingRequest().putEmpty().status shouldBe 401
       }
     }
 
     "return forbidden" when {
       "user with Agent affinity group and HMRC-AS-AGENT enrolment attempts to create mapping" in {
 
-        givenUserAuthorisedFor("HMRC-AS-AGENT", "AgentReferenceNumber", "TARN000003", "testCredId", agentCodeOpt = Some("TZRXXV"))
+        givenUserIsAuthorisedFor(
+          "HMRC-AS-AGENT",
+          "AgentReferenceNumber",
+          "TARN000003",
+          "testCredId",
+          agentCodeOpt = Some(agentCode))
         givenIndividualRegistrationExists(utr)
-        createMappingRequest(saOnlyIdentifier).putEmpty().status shouldBe 403
+        createMappingRequest().putEmpty().status shouldBe 403
       }
       "authenticated user with IR-SA-AGENT enrolment but without Agent Affinity group attempts to create mapping" in {
 
-        givenUserAuthorisedFor(`IR-SA-AGENT`, IRAgentReference, "2000000000", "testCredId", AffinityGroup.Individual, Some("TZRXXV"))
+        givenUserIsAuthorisedFor(
```

Comment:
This test also looks like it could be generalised from `IR-SA-AGENT` to any enrolment and moved to the general tests up at line 80.